app/models/shipit/deploy_spec/file_system.rb

```diff
@@ -70,11 +70,11 @@ def cacheable_config
             'variables' => deploy_variables.map(&:to_h),
             'max_commits' => maximum_commits_per_deploy,
             'interval' => pause_between_deploys,
-            'retries_on_timeout' => retries_on_deploy_timeout,
```

Comment:
Since it's a breaking yml config change, we'll want to reach out to or create PRs for the projects that have the current definition.

https://github.com/search?q=org%3AShopify+retries_on_timeout&type=code

Alternatively, we could keep supporting the old flag alongside the new.

Reply:
The list is small enough that I think I'll PR changes to each to use the new flag.